db/migrate/20180502155123_add_installed_deb.rb

```diff
@@ -0,0 +1,25 @@
+class AddInstalledDeb < ActiveRecord::Migration[4.2]
+  def change
+    create_table "katello_installed_debs" do |t|
+      t.timestamps
```

Comment:
We don't track the timestamps on Katello::InstalledPackage, do we need to here?

Reply:
removed.